tests/prague/eip7692_eof_v1/eip7069_extcall/test_calls.py

```diff
@@ -318,6 +321,71 @@ def test_eof_calls_eof_mstore(
     )
 
 
+identity = Address(0x04)
+blake2f = Address(0x09)
```

Comment:
All I need is for the precompile to fail, and blake2f seemed like one which fails the easiest. Can SHA256 fail at all?

Reply:
No it cannot. I guess this is good choice then.

Reply:
maybe what is missing is a comment (I can move the one from test function body up here for more visibility?)

Reply:
Blake has been on the deprecate for removal hitlist.  One of the EIP-2537 operations would be better, G1 add has a similar legth error.

Reply:
> Blake has been on the deprecate for removal hitlist. One of the EIP-2537 operations would be better, G1 add has a similar legth error.

This would preclude CancunEIP7692, which is still in use. I'll put it down to remember switching over. Thanks for flagging!